client/blocks/importer/wordpress/import-everything/index.tsx

```diff
@@ -239,21 +238,15 @@ export class ImportEverything extends SectionMigrate {
 		);
 	}
 
-	renderDefaultHoorayScreen() {
-		const { translate, stepNavigator } = this.props;
-
+	renderMigrationSuccessScreen() {
+		const { translate, targetSite, recordTracksEvent } = this.props;
 		return (
-			<div className="import__heading import__heading-center">
-				<Title>{ translate( 'Hooray!' ) }</Title>
-				<SubTitle>
-					{ translate( 'Congratulations. Your content was successfully imported.' ) }
-				</SubTitle>
-				<DoneButton
-					label={ translate( 'View site' ) }
-					onSiteViewClick={ () => {
-						this.props.recordTracksEvent( 'calypso_site_importer_view_site' );
-						stepNavigator?.goToSiteViewPage?.();
-					} }
+			<div className="import__heading import__heading-center migration-success">
+				<AsyncLoad
+					translate={ translate }
```

Comment:
On a second thought, would it be possible to skip passing this and simply use
```
const { __, hasTranslation } = useI18n();
```
directly in the `MigrationSuccess` component? Does it have some problems because of the AsyncLoad?

Reply:
Is there a reason/benefit to have it like you're suggesting instead of injected as props?

Reply:
It is usually better not to pass down props if it is not necessary and if the child component is able to simply call an hook 👍